### ISSUE TITLE
POST empty body request - 500 in @4.2.1

### ISSUE THREAD

**Issue by losikov** [cdimascio/express-openapi-validator#402]
**Describe the bug**
POST request with empty body. Expect to get 415 status code, and response body set in server.use((err, req, res, next) => {})
The issue is for 4.2.1, **works as expected in 4.2.0**.

**To Reproduce**
```
import request from 'supertest'
...
  it('should return 415 to a omitted request body', done => {
    request(server)
      .post('/api/v1/users/login')
      .set('Accept', 'application/json')
      .expect('Content-Type', /json/)
      .expect(415)
      .end(function(err, res) {
        console.log(res.body)
        if (err) return done(err)
        done()
      })
  })
```

**Actual behavior** 
Statuc code: 500
Content-Type: text/html; charset=utf-8
Body: {}

**Expected behavior**
Statuc code: 415
Content-Type: application/json
Body: set in server.use((err, req, res, next) => {})

**Comment by cdimascio** [comment]
@losikov thanks for the issue. Will have a look

**Comment by cdimascio** [comment]
This PR is the likely culprit. Feel free to have a look
https://github.com/cdimascio/express-openapi-validator/pull/398

**Comment by cdimascio** [comment]
@losikov fixed in`4.2.2`
